Horion/Module/Modules/FullBright.cpp

```diff
@@ -10,21 +10,25 @@ const char* FullBright::getModuleName() {
 	return "Fullbright";
 }
 
-void FullBright::onTick(C_GameMode* gm) {
-	if (gammaPtr != nullptr && *gammaPtr != 10) 
+void FullBright::onEnable() {
+	if (gammaPtr != nullptr) {
+		prevGamma = *gammaPtr;
```

Comment:
everythingunder prevGamma is unidentified or undeclared

Reply:
What do you mean?
gammaPtr gets defined in Hooks.cpp([1349](https://github.com/horionclient/Horion/blob/master/Memory/Hooks.cpp#L1349)) and i use a negative value to say that i don't  know the users gamma

Reply:
I was stupid as fuck and didn't add the .h file sorry.